### DISCUSSION TITLE
How to reduce the time affected by website access when deploying nuxt3 using PM2?

### DISCUSSION THREAD

**Discussion by mcpanl** [nuxt/nuxt#25088] [Questions]
### Environment

------------------------------
- Operating System: Windows_NT
- Node Version:     v18.12.1
- Nuxt Version:     3.9.0
- CLI Version:      3.10.0
- Nitro Version:    2.8.1
- Package Manager:  yarn@1.22.19
- Builder:          -
- User Config:      ssr, app, devtools, build, vite, experimental, modules, i18n, typescript, runtimeConfig, devServer, routeRules
- Runtime Modules:  @nuxt/ui@2.10.0, @nuxtjs/i18n@8.0.0, @nuxtjs/color-mode@3.3.2, @nuxt/image@1.1.0
- Build Modules:    -
------------------------------

### Reproduction

Any project can be replicated

### Describe the bug

I put my entire project on the server, and every time I need an update, I first `git pull`, then `yarn build`, and finally `pm2 restart myProject`. However, during the build process, the website may encounter a brief access error, indicating that some files in the `.output` folder do not exist. How can I achieve seamless updates or reduce the affected time.

I don't know if this is a feasible solution: for example, when building, first put all the files in the `.outputNew` folder, and then copy all the files in the `.outputNew` folder to the  `.output` folder and overwrite them after the build is completed.

Thanks! @danielroe

**Comment by xjccc** [comment]
i think its some cache at pm2

when you rebuild your project

you can try `pm2 stop` then `pm2 start`
